packages/zarf-agent/chart/Chart.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: v1
+description: Zarf agent
+name: zarf-agent
+version: 1.0.0
```

This _might_ align more to the zarf version
```suggestion
version: 0.51.0
```

---

I was matching [what was done with zarf-registry](https://github.com/zarf-dev/zarf/blob/19329edfb1b3e35515896920d39b9f6780dafc6d/packages/zarf-registry/chart/Chart.yaml#L4), but I agree that zarf-agent is more highly coupled with the actual zarf version